homeassistant/components/lifx/coordinator.py

```diff
@@ -156,3 +202,43 @@ async def async_set_color_zones(
                 apply=apply,
             )
         )
+
+    async def async_update_wifi_info(self) -> None:
+        """Get the bulb's wifi signal strength."""
+        _LOGGER.debug("Updating wifi signal strength for %s", self.device.label)
+        try:
+            wifi_info = await async_execute_lifx(self.device.get_wifiinfo)
+            self._rssi = int(math.floor(10 * math.log10(wifi_info.signal) + 0.5))
+        except asyncio.TimeoutError as ex:
+            raise UpdateFailed(
+                f"Failed to fetch RSSI sensor data from device: {self.device.ip_addr}"
+            ) from ex
+
+    async def async_update_hev_cycle(self) -> None:
```

Comment:
We shouldn't poll this unless at least one entity is enabled that needs it

Reply:
This is only available to HEV bulbs and they would all have it enabled by default, so if this is called, it's needed.

Reply:
Users can disable the entities